### PR TITLE
Move folder services/cincinnati-ci to app.ci

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-1658

/cc @alvaroaleman @stevekuznetsov

### CODE REVIEW

File: clusters/app.ci/supplemental-ci-images/cincinnati-ci/admin_rbac.yaml [hunk 1]
```diff
@@ -80,18 +68,6 @@ items:
...
         name: ci-image-sa
-  - kind: RoleBinding
-    apiVersion: rbac.authorization.k8s.io/v1
-    metadata:
-      name: cincinnati-ci-public-admin-binding-cluster
-      namespace: cincinnati-ci-public
-    roleRef:
-      apiGroup: rbac.authorization.k8s.io
-      kind: ClusterRole
-      name: admin
```
Comment (alvaroaleman) [inline clusters/app.ci/supplemental-ci-images/cincinnati-ci/admin_rbac.yaml:91]: Shouldn't this be the `pull-secret-namespace-manager`?

Comment (hongkailiu) [reply clusters/app.ci/supplemental-ci-images/cincinnati-ci/admin_rbac.yaml:91]: Updated.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**